src/core/Settings.js

```diff
@@ -699,7 +700,8 @@ function Settings() {
                 sid: null,
                 cid: null,
                 rtp: null,
-                rtpSafetyFactor: 5
+                rtpSafetyFactor: 5,
+                mode: Constants.CMCD_MODE_QUERY
```

Comment:
Add description for this parameter in JSDoc and add this option to the `index.d.ts`